ci/cases/gfsv17/obs_list.yaml

```diff
@@ -0,0 +1,17 @@
+observers:
+# ADT
+- !INC ${OBS_YAML_DIR}/adt_rads_all.yaml
+
+# SST
+- !INC ${OBS_YAML_DIR}/sst_avhrr_ma_l3u.yaml
+- !INC ${OBS_YAML_DIR}/sst_avhrr_mb_l3u.yaml
+- !INC ${OBS_YAML_DIR}/sst_avhrr_mc_l3u.yaml
+- !INC ${OBS_YAML_DIR}/sst_viirs_npp_l3u.yaml
+- !INC ${OBS_YAML_DIR}/sst_viirs_n20_l3u.yaml
+- !INC ${OBS_YAML_DIR}/sst_abi_g16_l3c.yaml
+- !INC ${OBS_YAML_DIR}/sst_abi_g17_l3c.yaml
+- !INC ${OBS_YAML_DIR}/sst_ahi_h08_l3c.yaml
+
+# Ice concentration
+- !INC ${OBS_YAML_DIR}/icec_amsr2_north.yaml
+- !INC ${OBS_YAML_DIR}/icec_amsr2_south.yaml
```

Comment:
Just an FYI, the `!INC` tag is going away.  I think its only in the ocean left.
There is no rush, I just wanted to drop a note.

Reply:
Thanks for the FYI!   @guillaumevernieres want to point this out to you 


Quick question, we also use this to include the defaults, how should this change to continue to include those?

Reply:
yes.  that is also on a `TODO` list.  I am not worried about the ocean stuff as it will be assembled using JCB